_docs_use-cases/use-case-transport-initiate-identitydeletionprocess.md

```diff
@@ -41,7 +41,6 @@ required_by:
 
 {% include properties_list.html %}
 
-The process of [Identity deletion]({% link _docs_integrate/delete-identities.md %}) can either be self-initiated or triggered via the Backbone Admin UI.
 In the case of [self-initiated Identity deletion]({% link _docs_integrate/delete-identities.md %}#self-initiated-identity-deletion), this use case can be utilized to initiate an [IdentityDeletionProcess]({% link _docs_integrate/data-model-overview.md %}#identitydeletionprocess) for your [Identity]({% link _docs_integrate/data-model-overview.md %}#identity).
```

Comment:
```suggestion
This use case can be utilized to initiate an [IdentityDeletionProcess]({% link _docs_integrate/data-model-overview.md %}#identitydeletionprocess) for your [Identity]({% link _docs_integrate/data-model-overview.md %}#identity).
```

Reply:
I am rephrasing the sentence, but I would like to keep the link to the scenario in any case. Please let me know if you don't like the new wording yet.